python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -1,23 +1,62 @@
+import json
+import logging
 from importlib.resources import files
 from typing import Sequence
 
-import datahub.emitter.mce_builder as mce_builder
-import datahub.metadata.schema_classes as schema_classes
-from datahub.configuration.common import GraphError
-from datahub.emitter.mce_builder import make_data_platform_urn
-from datahub.emitter.mcp import MetadataChangeProposalWrapper
-from datahub.ingestion.graph.client import DatahubClientConfig, DataHubGraph
-from datahub.ingestion.source.common.subtypes import (
+from data_platform_catalogue.client.datahub.graphql_helpers import (  # pylint: disable=E0611
+    parse_columns,
+    parse_created_and_modified,
+    parse_domain,
+    parse_owner,
+    parse_properties,
+    parse_relations,
+    parse_tags,
+)
+from data_platform_catalogue.client.datahub.search import (  # pylint: disable=E0611
+    SearchClient,
+)
+from data_platform_catalogue.client.exceptions import (  # pylint: disable=E0611
+    AspectDoesNotExist,
+    CatalogueError,
+    EntityDoesNotExist,
+    InvalidDomain,
+    ReferencedEntityMissing,
+)
+from data_platform_catalogue.entities import (  # pylint: disable=E0611
```

Comment:
I don't think these pylint directives should be there?

E0611 means the name can't be imported which would be a bug or environment issue https://pycodequ.al/docs/pylint-messages/e0611-no-name-in-module.html